### DISCUSSION TITLE
TERMUX AND VNC SESSION AUTOMATICALLY TERMINATING

### DISCUSSION THREAD

**Discussion by hasibr-126** [termux/termux-packages#18774] [Q&A]
I am trying to run Debian by tigervnc in termux using proot-distro. But, when i start the vnc server it closes after a while and termux also closes. But termux session doesn't  closes when i login to the distro,it only occurs in vnc.i think this is a vnc related problem.
![Screenshot_20231221_163350_Termux](https://github.com/termux/termux-packages/assets/121567729/af54c0bf-80c1-449a-a51d-6a256e3e0d53)

**Comment by twaik** [comment]
#!/bin/sh must be the first line of file.

**Reply by hasibr-126** [reply]
It didn't work then

**Comment by sylirre** [comment]
It closes due to [phantom process killer](https://github.com/agnostic-apollo/Android-Docs/blob/master/en/docs/apps/processes/phantom-cached-and-empty-processes.md#phantom-processes) or similar background process management mechanism of Android OS.
